crypto/fipsmodule/rand/internal.h

```diff
@@ -129,10 +129,30 @@ OPENSSL_INLINE int have_fast_rdrand(void) {
 #define MAX_BACKOFF_RETRIES 9
 OPENSSL_EXPORT void HAZMAT_set_urandom_test_mode_for_testing(void);
 
+// Final factor TBD
+#define PASSIVE_ENTROPY_WHITEN_FACTOR 10
```

Comment:
Is this a typo? What do you mean by whiten factor?

Reply:
Whitening is a technique/method to reduce bias in the distribution.